tests/components/sensor/test_recorder.py

```diff
@@ -3094,7 +3094,7 @@ def _sum(seq, last_state, last_sum):
                 sum_attributes,
                 seq[j : j + 1],
             )
-            start_meter = start + timedelta(minutes=1)
+            start_meter += timedelta(minutes=1)
```

Comment:
This was a bug in the test, it was inserting rows with the same timestamp down to the microsecond so selecting them back resulted in an indeterminate order